build.sbt

```diff
@@ -760,60 +737,3 @@ lazy val dlcTest = project
     coreJVM % testAndCompile,
     testkitCoreJVM
   )
-
-/** Given a database name, returns the appropriate
-  * Flyway settings we apply to a project (chain, node, wallet)
-  */
-def dbFlywaySettings(dbName: String): List[Setting[_]] = {
-  lazy val DB_HOST = "localhost"
-  lazy val DB_NAME = s"${dbName}.sqlite"
-  lazy val network = "unittest" //mainnet, testnet3, regtest, unittest
-
-  lazy val mainnetDir = s"${System.getenv("HOME")}/.bitcoin-s/mainnet/"
-  lazy val testnetDir = s"${System.getenv("HOME")}/.bitcoin-s/testnet3/"
-  lazy val regtestDir = s"${System.getenv("HOME")}/.bitcoin-s/regtest/"
-  lazy val signetDir = s"${System.getenv("HOME")}/.bitcoin-s/signet/"
-  lazy val unittestDir = s"${System.getenv("HOME")}/.bitcoin-s/unittest/"
-
-  lazy val dirs =
-    List(mainnetDir, testnetDir, regtestDir, signetDir, unittestDir)
-
-  //create directies if they DNE
-  dirs.foreach { d =>
```

Comment:
This is where the build config creates the directories automatically for us that leads us not to see errors that users my encounter